apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -27,7 +27,7 @@
 </head>
 <body>
   <header>
-    <nav class="navbar navbar-expand-md shadow-sm navbar-color navbar-<%= @user_configuration.navbar_type %>" aria-label="Navigation">
+    <nav class="navbar navbar-expand-md shadow-sm navbar-color navbar-<%= @user_configuration.navbar_type %>">
```

Comment:
`nav` tags already announce their role as Navigation, so there's no need for this label here.